build/Targets/VSL.Versions.targets

```diff
@@ -56,7 +59,7 @@
     <When Condition="'$(OfficialBuild)' == 'true' OR '$(SignedBuild)' == 'true'">
       <PropertyGroup>
         <AssemblyVersion>$(RoslynSemanticVersion).0</AssemblyVersion>
-        <BuildVersion>$(RoslynSemanticVersion).$(BuildNumberPart1)$(BuildNumberPart2)</BuildVersion>
+        <BuildVersion>$(RoslynSemanticVersion).0</BuildVersion>
```

Comment:
Doesn't this mean the file version no longer depends on the date? That's not good if it is.